### PR TITLE
Updated lighthouse CRD

### PR DESCRIPTION
**Pull request by aswinsuryan**
1)Changed the type of clusterIP from net.ip to string.

2)Changed the folder structure to match it with submariner CRD.

### CODE REVIEW

**Comment by aswinsuryan** [comment]
Updated the commit message and PR description.

**Comment by aswinsuryan** [comment]
> try to remove the merge commits, it makes the PR less clear.

Done

File: pkg/client/informers/externalversions/internalinterfaces/factory_interfaces.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ package internalinterfaces
 import (
 	time "time"
 
-	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	versioned "github.com/submariner-io/lighthouse//pkg/client/clientset/versioned"
```
Comment (mpeterson) [inline pkg/client/informers/externalversions/internalinterfaces/factory_interfaces.go]: extra forward slash
```suggestion
	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
```

File: pkg/client/informers/externalversions/factory.go [hunk 2]
```diff
@@ -23,9 +23,9 @@ import (
 	sync "sync"
 	time "time"
 
-	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	versioned "github.com/submariner-io/lighthouse//pkg/client/clientset/versioned"
```
Comment (mpeterson) [inline pkg/client/informers/externalversions/factory.go]: extra forward slash
```suggestion
	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
```

File: pkg/client/informers/externalversions/generic.go [hunk 3]
```diff
@@ -21,7 +21,7 @@ package externalversions
 import (
 	"fmt"
 
-	v1 "github.com/submariner-io/lighthouse/pkg/apis/multiclusterservice/v1"
+	v1 "github.com/submariner-io/lighthouse//pkg/apis/lighthouse.submariner.io/v1"
```
Comment (mpeterson) [inline pkg/client/informers/externalversions/generic.go]: extra forward slash
```suggestion
	v1 "github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v1"
```

File: pkg/client/informers/externalversions/lighthouse.submariner.io/v1/multiclusterservice.go [hunk 4]
```diff
@@ -21,10 +21,10 @@ package v1
 import (
 	time "time"
 
-	multiclusterservicev1 "github.com/submariner-io/lighthouse/pkg/apis/multiclusterservice/v1"
-	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	lighthousesubmarineriov1 "github.com/submariner-io/lighthouse//pkg/apis/lighthouse.submariner.io/v1"
+	versioned "github.com/submariner-io/lighthouse//pkg/client/clientset/versioned"
```
Comment (mpeterson) [inline pkg/client/informers/externalversions/lighthouse.submariner.io/v1/multiclusterservice.go]: extra forward slash
```suggestion
	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
```

File: pkg/client/informers/externalversions/lighthouse.submariner.io/v1/multiclusterservice.go [hunk 5]
```diff
@@ -21,10 +21,10 @@ package v1
...
+	v1 "github.com/submariner-io/lighthouse//pkg/client/listers/lighthouse.submariner.io/v1"
```
Comment (mpeterson) [inline pkg/client/informers/externalversions/lighthouse.submariner.io/v1/multiclusterservice.go]: extra forward slash
```suggestion
	v1 "github.com/submariner-io/lighthouse/pkg/client/listers/lighthouse.submariner.io/v1"
```

File: pkg/client/informers/externalversions/lighthouse.submariner.io/v1/multiclusterservice.go [hunk 6]
```diff
@@ -21,10 +21,10 @@ package v1
 import (
 	time "time"
 
-	multiclusterservicev1 "github.com/submariner-io/lighthouse/pkg/apis/multiclusterservice/v1"
-	versioned "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	lighthousesubmarineriov1 "github.com/submariner-io/lighthouse//pkg/apis/lighthouse.submariner.io/v1"
```
Comment (mpeterson) [inline pkg/client/informers/externalversions/lighthouse.submariner.io/v1/multiclusterservice.go]: extra forward slash
```suggestion
	lighthousesubmarineriov1 "github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v1"
```

File: pkg/client/listers/lighthouse.submariner.io/v1/multiclusterservice.go [hunk 7]
```diff
@@ -19,7 +19,7 @@ limitations under the License.
 package v1
 
 import (
-	v1 "github.com/submariner-io/lighthouse/pkg/apis/multiclusterservice/v1"
+	v1 "github.com/submariner-io/lighthouse//pkg/apis/lighthouse.submariner.io/v1"
```
Comment (mpeterson) [inline pkg/client/listers/lighthouse.submariner.io/v1/multiclusterservice.go]: extra forward slash 
```suggestion
	v1 "github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v1"
```

**Comment by mpeterson** [comment]
Thanks @aswinsuryan for taking care of these changes

**Pull request merged**